packages/test/test-end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -89,33 +89,42 @@ describeFullCompat("Flushing ops", (getTestObjectProvider) => {
     let dataObject2map1: SharedMap;
     let dataObject2map2: SharedMap;
 
-    beforeEach(async () => {
+    async function setupContainers(runtimeOptions?: IContainerRuntimeOptions) {
+        const configCopy = { ...testContainerConfig, runtimeOptions };
+
         // Create a Container for the first client.
-        container1 = await provider.makeTestContainer(testContainerConfig) as Container;
+        container1 = await provider.makeTestContainer(configCopy) as Container;
         dataObject1 = await requestFluidObject<ITestFluidObject>(container1, "default");
         dataObject1map1 = await dataObject1.getSharedObject<SharedMap>(map1Id);
         dataObject1map2 = await dataObject1.getSharedObject<SharedMap>(map2Id);
 
         // Load the Container that was created by the first client.
-        const container2 = await provider.loadTestContainer(testContainerConfig);
+        const container2 = await provider.loadTestContainer(configCopy);
         dataObject2 = await requestFluidObject<ITestFluidObject>(container2, "default");
         dataObject2map1 = await dataObject2.getSharedObject<SharedMap>(map1Id);
         dataObject2map2 = await dataObject2.getSharedObject<SharedMap>(map2Id);
 
+        // @ts-expect-error older versions rely on the "setFlushMode" method
```

Comment:
@andre4i Since older versions rely on the `setFlushMode` method I had to do this sort of hacky workaround. Do you know of a better way, or should we delay this PR until your code is in N-2?

Reply:
I think that because we're in `next`, the N-2 thing doesn't apply anymore in terms of breaking back-compat. Just make sure you don't forget about this and track the removal in the cleanup work item. You can also add a todo comment and mention the work item number.